src/env-var/cc-env-var-input.js

```diff
@@ -124,17 +123,16 @@ export class CcEnvVarInput extends LitElement {
 
   static get styles () {
     return [
-      defaultThemeStyles,
       skeletonStyles,
       // language=CSS
       css`
         :host {
           --cc-gap: 0.5rem;
+          color: var(--cc-color-text-default);
           display: block;
         }
 
         .name {
-          background-color: #fff;
```

Comment:
may seem weird but this is to avoid this:
![image](https://user-images.githubusercontent.com/100240294/177960093-79c6097c-55a9-41bc-98b8-f60c32843032.png)

Labels on the left should just be texts with no background.